ENH Adds Poisson criterion in RandomForestRegressor #19304 (#19836)
scikit-learn/scikit-learn: doc/whats_new/v1.0.rst

  output which is used to initialize the code.
  :pr:`18433` by :user:`Bruno Charron <brcharron>`.

> - |Enhancement| Updated :func:`fit` in :class:`BaseForest` to provide official
>   support of the Poisson criterion for RandomForestRegressor. A corresponding
>   test case was also included. :pr:`19836` by :user:`Brian Sun <bsun94>`

Reviewer: ```suggestion
- |Enhancement| Documented and tested support of the Poisson criterion
  for :class:`ensemble.RandomForestRegressor`.
  :pr:`19836` by :user:`Brian Sun <bsun94>`
```
In addition, this has to be put **under** the section :mod:`sklearn.ensemble` and not before.

Author: Apologies for that - silly misplacement. Will fix - thanks!
